setup.py

```diff
@@ -1,39 +1,28 @@
 #!/usr/bin/env python
 """Setup file for python-matlab-bridge"""
 
-import platform
 import os
 import sys
 import shutil
+import glob
+from distutils.core import setup
```

Comment:
The comment below says `distutils` was imported later on purpose. (I'm not really familiar with why though).